docs/spl-docs/sort-command.md

```diff
@@ -0,0 +1,94 @@
+
+# sort Command
+
+## Description
+
+The `sort` command organizes the output based on the designated fields. If a result lacks a specified field, it is considered to have either the minimum or maximum value for that field, depending on whether the sorting is in ascending or descending order.
+
+When the initial argument to the `sort` command is a numeral, it limits the output to that number of results, presented in sequence. In the absence of a specified number, a default cap of 10000 results applies. Specifying the number 0 as the first argument returns all results.
+
+## Syntax
+The required syntax is in bold
+
+**sort**
+[\<count\>]
+**\<sort-by-clause\>...**
+[desc]
+
+### Required Arguments
+
+#### \<sort-by-clause\>
+**Syntax:** `[ - | + ] <sort-field>, ( - | + ) <sort-field> ...`\
+**Description:** Specifies the fields to sort by, including the order of sorting. A minus sign (-) indicates descending order, while a plus sign (+) denotes ascending order. For multiple fields, separate each with a comma.
+
+### Optional Arguments
+
+#### \<count\>
+**Syntax:** `<int> | limit=<int>`\
+**Description:** Determines the number of results to display after sorting. Absent a specified count, a default threshold of 10000 results is applied. Specifying 0 returns all possible results. The count can be directly indicated by an integer or introduced with the prefix 'limit=', e.g., limit=10.
+
+**Default:** 10000
+
+#### desc
+**Syntax:** `d | desc`\
+**Description:** Inverts the sequence of the results. When multiple fields are designated, it inversely arranges the values within those fields according to their specified sequence. For instance, with three fields indicated, the `desc` parameter inversely organizes the values in the first field. For any group of identical values in the first field, it similarly inverts the order of related values in the second field, and for each group of matching values in the second field, it reverses the order of associated values in the third field.
+
+### Sort Field Options
+
+#### \<sort-field\>
+**Syntax:** `<field> | auto(<field>) | str(<field>) | ip(<field>) | num(<field>)`\
```

Comment:
nit: change `|` to "or". See https://github.com/siglens/siglens-docs/pull/95#discussion_r1674394930

Same comment some other places